Pidaq/Examples/can-receiver.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+import can
+
+# Function Definitions
+
+def create_can_filters(raw_filters):
+    """ Creates a list of can filter objects
+
+    Parameters:
+    raw_filters (list): list of message filter objects as defined in the json schema
+
+    Returns:
+    can_filters (list): list of python-can filter objects
+    """
+    can_filters = []
+    for raw_can_filter in raw_can_filters:
+        can_id = raw_can_filter["canId"]
+        can_mask = raw_can_filter["canMask"]
+        extended = raw_can_filter["extended"]
+        can_filters.append({
+            "can_id": can_id,
+            "can_mask": can_mask,
+            "extended": extended
+        })
+    return can_filters
+
+if __name__ == "__main__":
+    json_config = open('receiverInfo.json', mode='r')
+    can_config = json.load(json_config)
+    can_bus = can.Bus(bustype=can_config["canInfo"]["busType"],
+                      channel=can_config["canInfo"]["serialPort"],
+                      bitrate=can_config["canInfo"]["bitrate"])
+    raw_can_filters = [
```

Comment:
not sure if this is proper styling but starting a list bracket on a line, writing code on the next and closing it in the second line looks weird-shouldn't it either be both brackets on the same line or opening bracket, code on the next line and then the closing bracket on another new line?